src/cuda_kernels/GistCudaSetup.cu

```diff
@@ -0,0 +1,239 @@
+#include "GistCudaSetup.cuh"
+#include "GistCudaCalc.cuh"
+#include <iostream>
```

Comment:
I had to change this from `<iostream>` to `<cstdio>` in order to avoid:
```
GistCudaSetup.cu(195): error: identifier "printf" is undefined

1 error detected in the compilation of "/tmp/tmpxft_00002bc7_00000000-5_GistCudaSetup.cpp4.ii".
```
on my local machine (gcc 5.4, cuda 8)

Reply:
I changed the `<iostream>` to `<cstdio>` and moved the `#include <cstdio>` into the GistCudaSetup.cuh. Should now work, but could you please double check?